config/vite.vue.dev.config.js

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from 'vite'
+
+import viteReactConfig from './vite.vue.config'
```

Comment:
I know that the correct config is imported, but please rename this to `viteVueConfig`